README.md

```diff
@@ -254,6 +254,12 @@ ComfyJS.Disconnect();
  - **onHosted**`( user, viewers, autohost, extra )`
     - Responds to channel being hosted
     - Requires being authorized as the broadcaster
+ - **onBan**`( bannedUsername, reason, extra )`
+    - Responds to a user being banned
+    - reason will be null unless authorized as broadcaster or moderator for the channel
+ - **onTimeout**`( timedOutUsername, reason, durationInSeconds, extra )`
+    - Responds to a user being timed out
+    - reason will be null unless authorized as broadcaster or moderator for the channel
```

Comment:
Same as above - I think `reason` will always be `null`.

Reply:
Ahh, great catch! I will fix and resubmit. Thank you!!

Reply:
Okay pulled out the reason param and README is updated accordingly. Thanks again Ben!